com.unity.multiplayer.mlapi/Tests/Runtime/MultiInstance/RPCTests.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections;
+using System.Diagnostics;
+using System.Linq;
+using MLAPI.Messaging;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.TestTools;
+using Debug = UnityEngine.Debug;
+
+namespace MLAPI.RuntimeTests
+{
+    public class RPCTests
+    {
+        public class RPCTestNetworkBehaviour : NetworkBehaviour
+        {
+            public event Action OnServer_RPC;
+            public event Action OnClient_RPC;
+
+            [ServerRpc]
+            public void MyServerRpc()
+            {
+                OnServer_RPC();
+            }
+
+            [ClientRpc]
+            public void MyClientRpc()
+            {
+                OnClient_RPC();
+            }
+        }
+
+        [UnityTest]
+        public IEnumerator TestRPCs()
```

Comment:
really cool. I tried reusing this in NetworkTransform tests and it works great. It feels like a lot of this should be put in a "Setup()" UnitySetUp method though. It'd make it easier to reuse with multiple test cases. And by doing this, you'll notice you have a Create, but no Destroy method for your helper. (this could be done in a followup PR for sure)